src/JWT.php

```diff
@@ -399,13 +399,28 @@ public static function jsonEncode(array $input): string
      * @throws InvalidArgumentException invalid base64 characters
      */
     public static function urlsafeB64Decode(string $input): string
+    {
+        return \base64_decode(self::convertBase64urlToBase64($input));
+    }
+
+    /**
+     * Convert a string in the base64url (URL-safe Base64) encoding to standard base64.
+     *
+     * @param string $input A Base64 encoded string with URL-safe characters (-_ and no padding)
+     *
+     * @return string A Base64 encoded string with standard characters (+/) and padding (=), when
+     * needed.
+     *
+     * @see https://www.rfc-editor.org/rfc/rfc4648
+     */
+    public static function convertBase64urlToBase64(string $input): string
```

Comment:
This is a nit, but to be more consistent with our other namings, we should capitalize `Url` here:
```suggestion
    public static function convertBase64UrlToBase64(string $input): string
```